src/corelibs/U2View/src/ov_mca/McaEditor.h

```diff
@@ -81,6 +82,11 @@ class U2VIEW_EXPORT McaEditor : public MaEditor {
         return gotoSelectedReadAction;
     }
 
+    MaEditorWgt* getMaEditorWgt(uint index = 0) const override {
```

Comment:
Please replace all uints for indexes to ints in the code. This should be as simple as 'replace-all' in the editor.